core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -432,26 +432,23 @@ private void checkTempBlockOwnedBySession(long sessionId, long blockId)
    */
   private void abortBlockInternal(long sessionId, long blockId) throws BlockDoesNotExistException,
       BlockAlreadyExistsException, InvalidWorkerStateException, IOException {
-    long lockId = mLockManager.lockBlock(sessionId, blockId, BlockLockType.WRITE);
```

Comment:
the lock is needed when the client issues multiple abort command concurrently. @apc999 what do you think?